packages/mui-material/src/Switch/Switch.js

```diff
@@ -224,6 +237,7 @@ const Switch = React.forwardRef(function Switch(inProps, ref) {
         checkedIcon={icon}
         ref={ref}
         ownerState={ownerState}
+        color={color}
```

Comment:
```suggestion
```
No need to pass color as a prop here, the `ownerState` is enough.

Reply:
legacy of the debug session 🙈